app/src/main/res/values/styles.xml

```diff
@@ -21,4 +21,20 @@
         <item name="android:textSize">14sp</item>
         <item name="android:textColor">@color/colorAccent</item>
     </style>
+
+    <style name="RowItemName">
```

Comment:
There should be a `<style name="RowItem"></style>` where you define common attributes like color being white and text style being bold.
Then the other three should be RowItem.Name, RowItem.Category, RowItem.Type where you define text size individually